pr-scripts/create_pr.sh

```diff
@@ -23,19 +23,31 @@ OLD_TAG="$2"
 NEW_TAG="$3"
 FILEPATH="$4"
```

Comment:
Let's define $5 here too, I think it should work

Reply:
Done!